apps/super-app/src/common/components/More.tsx

```diff
@@ -0,0 +1,6 @@
+const More = () => {
+    return <h1>More Page placeholder</h1>;
+  };
+  
+  export default More;
+  
```

Comment:
There are a few formatting errors like this. It's pretty easy to fix (just use `make format`). You can check the CI for more info https://github.com/bettersg/call-home/pull/103/checks.